tests/js/app/modules/layout/testPaginatedScrolling.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016 Endless Mobile, Inc.
+
+const Gtk = imports.gi.Gtk;
+
+const Utils = imports.tests.utils;
+Utils.register_gresource();
```

Comment:
Yea it is because the paginatedScrolling layout uses a `.ui` template file, which has to get compiled into the gresource